fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionVote.java

```diff
@@ -34,15 +28,13 @@ public class ElectionVote {
    */
   public ElectionVote(
       String questionId,
-      List<Integer> vote,
+      Integer vote,
```

Comment:
Why do we updated this in this new json schema ?

Reply:
The vote contain now an integer if OPEN_BALLOT, or a String id SECRET_BALLOT. The vote is unique, however, the list is kept we add vote with multiple options.

Reply:
Same question, would this be a problem if we have multiple questions?